ci/appveyor-cpp-setup.bat

```diff
@@ -86,12 +86,12 @@ if defined need_vcvarsall (
 @rem
 @rem Use clcache for faster builds
 @rem
-pip install -q git+https://github.com/frerich/clcache.git
```

Comment:
Couldn't we just install this always using conda?

Reply:
Is there an up-to-date conda package?

Reply:
Oh yeah. Working on it.

Reply:
I see that conda-forge has version 4.1.0, which is quite old.

Reply:
When do you intend the new version to be available on conda-forge? It would be nice to merge this PR quickly.